composer.json

```diff
@@ -35,6 +35,9 @@
         "escapestudios/symfony2-coding-standard": "^3.9",
         "squizlabs/php_codesniffer": "^3.5"
     },
+    "suggest": {
+        "symfony/mailer": "If you want to send emails using this modern mailer from Symfony"
```

Comment:
Would not add this. Composer suggestions are used way to much imo and I think it is not relevant enough to display it on every composer install.